servers/src/main/java/tachyon/master/next/block/BlockMaster.java

```diff
@@ -322,6 +329,7 @@ private void processWorkerRemovedBlocks(MasterWorkerInfo workerInfo,
       masterBlockInfo.removeWorker(workerInfo.getId());
       if (masterBlockInfo.getNumLocations() == 0) {
         mLostBlocks.add(removedBlockId);
+        mBlocks.remove(removedBlockId);
```

Comment:
hrmmm, I think we might need to keep this information around, since this has length information about the block.